README.md

```diff
@@ -1,3 +1,9 @@
 # rfcs
 
-Request for changes, new features, hints, etc. for webhint.
+🛑🛑🛑🛑🛑
+
+**This repo is now deprecated.**
+
+For changes, new features, hints, etc. please [use this link](https://github.com/webhintio/hint/issues/new)**
```

Comment:
Need to include `/choose` at the end of the URL to prompt which type of issue to create based on templates. Otherwise this lands the user at the generic "New Issue" form.

```suggestion
For changes, new features, hints, etc. please [use this link](https://github.com/webhintio/hint/issues/new/choose)**
```